app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+# Create A New Web Application To Manage Nimbus Experiments
+
+* Status: proposed
+* Deciders: Kate Hudson, Jared Lockhart, TBD
+* Date: 2020-08-18
+
+## Context and Problem Statement
+
+Experimenter is a web application developed solely for the purpose of managing product experiments at Mozilla.  It was originally designed to manage the Shield experimentation workflow, and has expanded to include a comprehensive workflow for managing Normandy experiments.  Project Nimbus is being developed with very different core assumptions than the previous experimentation platforms, and it is unclear whether building on top of the existing Experimenter infrastructure or creating a new purpose specific application will best suit the project's needs.
+
+## Decision Drivers <!-- optional -->
+
+* Meeting Project Nimbus' needs in a rapid, continuous fashion with minimal downtime
+* Contributability by a large, distributed group of people with different backgrounds (web engineers, data engineers, data scientists)
+* Fitting cohesively into the FXA engineering pipeline/culture
+* Continuing to support the existing experimentation (Normandy) workflow
```

Comment:
I think something we need to add as a decision driver is related to who will be maintaining the tooling long term.

If there is any chance that the data org/data tools team would be part of the long-term maintenance, it's worth considering what type of tooling we work with in our pipelines. Generally we work with Python/Svelte.

Reply:
Unfortunately AFAIK we don't yet know who will have long term ownership.